android/engine/src/main/java/org/smartregister/fhircore/engine/rulesengine/RulesFactory.kt

```diff
@@ -565,6 +566,22 @@ constructor(
       }
       return serviceStatus
     }
+
+    fun generateListTaskServiceStatus(tasks: List<Task>): String {
```

Comment:
Can you harmonize this service function with this similar function [`RulesFactory#generateTaskServiceStatus`](https://github.com/opensrp/fhircore/blob/79b564eb2e8f475d18a1694cb1c50ef84cb19295/android/engine/src/main/java/org/smartregister/fhircore/engine/rulesengine/RulesFactory.kt#L544)